src/js/utils.js

```diff
@@ -78,7 +78,7 @@ export const validateForm = () => {
     if (fielValid[idField].required && !value) {
       field.classList.add("error");
       spanError.classList.add("span_error");
-      spanError.innerHTML =
+      spanError.innerHTML = "<i class=\"fa fa-exclamation-triangle\">&nbsp&nbsp</i>" +
         "Please enter the required field";
```

Comment:
You could use this to avoid `\"fa fa-exclamation-triangle\"`
```suggestion
      spanError.innerHTML = `<i class="fa fa-exclamation-triangle">&nbsp&nbsp</i> Please enter the required field`;
```

Reply:
Done. I updated it.